### PR TITLE
Added toggle on/off "permissive search" on Clients name (IP-544)

### PR DESCRIPTION
**Pull request by musapinar**
A better integration of the "button" / UI improvement would be appreciated. I'm struggling with Bootstrap columns/labels/form-group
Pushing pull request regardless.

### CODE REVIEW

File: application/language/english/ip_lang.php [hunk 1]
```diff
@@ -175,6 +175,7 @@
     'email_templates'                              => 'Email Templates',
     'enabled'                                      => 'Enabled',
     'enable_debug_mode'                            => 'Enable the Debug Mode',
+    'enable_permissive_search_Clients'             => 'Enable permissive search',
```
Comment (Kovah) [inline application/language/english/ip_lang.php]: the key should be all-lowercase

**Comment by Kovah** [comment]
Could you change the branch to `1.5.4` instead of master?

**Comment by aalwash** [comment]
And don't make PR to the master itself

@Kovah do we want this to be in 1.5.6 or 1.6?
So he can make a PR to that branch

https://github.com/InvoicePlane/InvoicePlane/blob/master/CONTRIBUTING.md

**Comment by Kovah** [comment]
@musapinar Could you also change the HTML IDs? See the review above for more details. :)

**Comment by Kovah** [comment]
Thanks for oyur work @musapinar ! :+1:

**Pull request merged**